minio/scripts/file-operations.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+# Executable that waits for a port at an address to be open.
+# For further details go to: https://github.com/ufoscout/docker-compose-wait
+/wait
+
+echo "Add s3 location alias called storage to minio config"
+mc config host add storage http://minio:9000 longkey verysecretkey
+
+echo "Create private bucket (private by default)"
+mc mb storage/private-bucket
+
+echo "Upload file to private bucket"
+mc cp ./assets/file.txt storage/private-bucket
+
+echo "Get meta data about uploaded file"
+mc stat storage/private-bucket/file.txt
```

Comment:
misses the similar: store arbitrary object tags onto a file

Reply:
should be there now.